Extensions/Manage/Dnn.PersonaBar.Users/Components/Dto/UserFilters.cs

```diff
@@ -7,6 +7,9 @@ public enum UserFilters
         Deleted = 2,
         SuperUsers = 3,
         RegisteredUsers = 4,
-        All = 5
```

Comment:
Should the value of this Enum have changed to 8 from 5? Not sure if this is used anywhere else, but this seems like it may cause some issues or potential bugs. Would it have been better to leave All as 5 and added the new items after?

Reply:
I thought about this and this was my reasoning:

1. when the filter is being requested through WebAPI it is serialised in order, so all would be somewhere in the middle

2. It would mean someone took a dependency *on the Users PersonaBar extension*. It's not a core enum. I don't know of anyone who would do something foolish like that. It would also surprise me that anyone who would live that dangerously would expect that to remain stable.